backend/models/generic.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 type Common struct {
-	ID        uint           `gorm:"primaryKey" json:"id"`
+	Id        uint           `gorm:"primaryKey" json:"id"`
 	CreatedAt time.Time      `json:"created_at"`
 	UpdatedAt time.Time      `json:"updated_at"`
 	DeletedAt gorm.DeletedAt `gorm:"index" json:"deleted_at"`
```

Comment:
What's use-case of this column? I understand how CreatedAt and UpdatedAt could be useful for most types but when we delete something we're just going to delete that row aren't we?

Reply:
I removed it in https://github.com/lynx-locks/lynx-web/pull/19, but I can add it back if there is value.